### ISSUE TITLE
balancemagic.cfg missing

### ISSUE THREAD

**Issue by brac3r** [DiscoveryGC/FLHook#190]
The Balance Magic Plugin is missing the config file.

balancemagic\Main.cpp line 75 
`string scPluginCfgFile = string(szCurDir) + "\\flhook_plugins\\balancemagic.cfg";`

I'm not familiar with C++, but I'm pretty sure this LoadSetting function is trying set a bunch of damage multipliers. Should there not be a balancemagic.cfg? There are plenty of other config files, that one doesn't exist.

Thanks in advance.

**Comment by dd84ai** [comment]
yeah, pretty much all server sides configs aren't updated at client side.
they are updated only at server side.

you know, for the purpose of easier collaboration, we should probably encourage other guys putting if possible those files for update. But all files will not be possible to keep updated because some of then are having sensitive information.

ergh, ideally you could try joining officially Coding devs of Freelancer Discovery and then u'll have access to them as it is.

**Comment by dd84ai** [comment]
TLDR: not a bug, but a feature ^_^ the developers from category `admins` should be approving putting not sensitive ones for update. Make to them request at forum or at Discovery discord server in admin lounge (or both)

**Comment by dd84ai** [comment]
@brac3r checked the file balancemagic.cfg
it does not have any sensetive information, so here it is update to it state
```
[DamageAdjust]
; munition = LF/HF/VHF, SHF/Bmb/Fr, Trns, Gnbt, Crsr, Bcrs, Btls, Solar

; Corona
torpedo01_mark022_ammo = 1, 1, 3.0, 3.0, 4.5, 7.0, 9.0

; Claymore Beam Cannon
bw_shf_forward_gun01_ammo = 1, 1, 3.5, 6.0, 8.0, 8.0, 8.0
; Claymore Plasma Accelerator
bw_shf_forward_gun02_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5

; SNACs
;event_snova_cultist_ammo = 0.136, 1
dsy_nomad_torpedo01_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_aux_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_ai_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_bhg_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_bretunlaw_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_bretlaw_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_civ_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_classic_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_coalition_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_core_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_corsairs_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_cr_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_gaunlaw_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_galaw_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_hf_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_kusunlaw_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_kuslaw_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_libunlaw_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_liblaw_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_order_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_outcasts_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_reaper_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_rheinunlaw_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_rheinlaw_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_wild_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
dsy_snova_xeno_ammo = 1, 2.0, 5.0, 8.0, 9.5, 9.5, 9.5
```
